parsers/to-jss-custom-properties/tokens/textStyle.ts

```diff
@@ -0,0 +1,38 @@
+import { TextStyleToken } from '../../../types';
+import { FormatTokenType } from '../to-jss-custom-properties.parser';
+import tinycolor from 'tinycolor2';
+
+export class TextStyle extends TextStyleToken {
+  constructor(token: Partial<TextStyleToken>) {
+    super(token);
+  }
+
+  toJss({ textStyleFormat = 'classObject', colorFormat = 'hex', fontSizeUnit = 'px' }: FormatTokenType) {
+    const { font, color: c, size: fs, lineHeight: lh } = this.value;
+    const { fontPostScriptName, fontWeight } = font.value;
+    const color = c.value;
+    const { measure: fontSizeRaw } = fs.value;
+    const { measure: lineHeight, unit: lhUnit } = lh.value;
+    const fontSize = {
+      px: Math.round((fontSizeRaw * 4) / 3),
```

Comment:
The default font size from the API will be in `px`. You should convert from `px` to `pt`.